specification/recoveryservices/Recoveryservices.Management/models.tsp

```diff
@@ -0,0 +1,1685 @@
+ */
+model ClientDiscoveryValueForSingleApi {
+  /**
+   * Name of the Operation.
+   */
+  name?: string;
+
+  /**
+   * Contains the localized display information for this particular operation
+   */
+  display?: ClientDiscoveryDisplay;
+
+  /**
+   * The intended executor of the operation;governs the display of the operation in the RBAC UX and the audit logs UX
+   */
+  origin?: string;
+
+  /**
+   * ShoeBox properties for the given operation.
+   */
+  properties?: ClientDiscoveryForProperties;
+}
+
+/**
+ * Localized display information of an operation.
+ */
+model ClientDiscoveryDisplay {
+  /**
+   * Name of the provider for display purposes
+   */
+  provider?: string;
+
+  /**
+   * ResourceType for which this Operation can be performed.
+   */
+  resource?: string;
+
+  /**
+   * Operations Name itself.
+   */
+  operation?: string;
+
+  /**
+   * Description of the operation having details of what operation is about.
+   */
+  description?: string;
+}
+
+/**
+ * Class to represent shoebox properties in json client discovery.
+ */
+model ClientDiscoveryForProperties {
+  /**
+   * Operation properties.
+   */
+  serviceSpecification?: ClientDiscoveryForServiceSpecification;
+}
+
+/**
+ * Class to represent shoebox service specification in json client discovery.
+ */
+model ClientDiscoveryForServiceSpecification {
+  /**
+   * List of log specifications of this operation.
+   */
+  logSpecifications?: ClientDiscoveryForLogSpecification[];
+}
+
+/**
+ * Class to represent shoebox log specification in json client discovery.
+ */
+model ClientDiscoveryForLogSpecification {
+  /**
+   * Name of the log.
+   */
+  name?: string;
+
+  /**
+   * Localized display name
+   */
+  displayName?: string;
+
+  /**
+   * Blobs created in customer storage account per hour
+   */
+  blobDuration?: string;
+}
+
+/**
+ * Patch Resource information, as returned by the resource provider.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model PatchVault extends PatchTrackedResource {
+  /**
+   * Properties of the vault.
+   */
+  properties?: VaultProperties;
+
+  /**
+   * Identifies the unique system identifier for each Azure resource.
+   */
+  sku?: Sku;
+
+  /**
+   * Identity for the resource.
+   */
+  identity?: IdentityData;
+}
+
+/**
+ * Tracked resource with location.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model PatchTrackedResource extends Azure.ResourceManager.Foundations.Resource {
+  /**
+   * Resource location.
+   */
+  location?: string;
+
+  /**
+   * Resource tags.
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  tags?: Record<string>;
+
+  /**
+   * Optional ETag.
+   */
+  etag?: string;
```

Comment:
[ARMBlockingFeedback] why not using common types?

Reply:
I think you are talking about the TypeSpec Envolope model `EntityTagProperty`? If so, it is `eTag`. `T` is upper case.

Reply:
So you mean it would be a breaking change to use common types then? OK, need ot figure out what to do here :)

Reply:
So unfortunate they use the wrong property name. Can you start tracking all these arm resource property violations where it'd be breaking to fix them, in more detail, to drive the next round of improvements?

Ideally there should be no Azure services that have these resource property violations, and I think our leaders would benefit from having detailed list available of what the violations are.